lib/config/locales/en.yml

```diff
@@ -31,6 +31,17 @@ en:
             since the document did not actually get saved."
           resolution: "Double check all before callbacks to make sure they are
             not unintentionally returning false."
+        create_collection:
+          message: "Cannot create collection %{collection_name}
+            with options %{collection_options}. The following error was raised:
+            %{server_error}."
```

Comment:
This should contain both the exception class and the exception message following the `"#{e.class}: #{e}"` pattern  we normally use.

Reply:
I implemented this in the exception constructor, I believe this is what you suggested.